staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -5488,6 +5500,15 @@ type PodLogOptions struct {
 	// the actual log data coming from the real kubelet).
 	// +optional
 	InsecureSkipTLSVerifyBackend bool `json:"insecureSkipTLSVerifyBackend,omitempty" protobuf:"varint,9,opt,name=insecureSkipTLSVerifyBackend"`
+
+	// Return the specific log stream. Defaults to "all".
```

Comment:
Let's improve this godoc by describing the action in greater detail:

```
Specify which container log streams to return to the client. By default both stdout and stderr are returned interleaved.  <describe what happens on Windows>.
Acceptable values are:
This is an alpha field ....
```

Reply:
> <describe what happens on Windows>.

I think this feature should make no difference on windows 🤔

Reply:
Do windows containers have stderr and stdout?

Reply:
Yeah I believe so https://docs.microsoft.com/en-us/virtualization/windowscontainers/troubleshooting#docker-container-logs IIUC.